servant-foreign/src/Servant/Foreign/Internal.hs

```diff
@@ -295,6 +295,13 @@ instance HasForeign lang sublayout => HasForeign lang (Vault :> sublayout) where
   foreignFor lang Proxy req =
     foreignFor lang (Proxy :: Proxy sublayout) req
 
+instance HasForeign lang sublayout =>
+  HasForeign lang (WithNamedConfig name config sublayout) where
```

Comment:
Doesn't there need to be an instance for `(WithNamedConfig name config s) :> sublayout` too?